enterprise/frontend/src/embedding-sdk/components/private/SdkError/SdkError.tsx

```diff
@@ -0,0 +1,52 @@
+import { useSdkSelector } from "embedding-sdk/store";
+import { getErrorComponent } from "embedding-sdk/store/selectors";
+import type {
+  LoginStatusError,
+  SdkErrorComponentProps,
+} from "embedding-sdk/store/types";
+import { Box, Center, Icon, Stack, Text } from "metabase/ui";
+
+import { getErrorInfo } from "./errors";
+
+export const SdkError = ({ status, code }: LoginStatusError["data"]) => {
```

Comment:
This is our internal `SdkError` component so all we need is the status. The custom error will have all of the information available in case people want to use it.